.circleci/config.yml

```diff
@@ -6,7 +6,7 @@ parameters:
     default: dwave-tabu
 
 orbs:
-  ocean: dwave/ocean@1.1
+  ocean: dwave/ocean@1.2
```

Comment:
Doesn't really matter, but would it be easier to just do `dwave/ocean@1` so that we don't need to update these when the orb gets updated?

Reply:
I omitted the patch version for that reason exactly (so fixes would be pulled automatically).

Wanted to start with explicit minor, just to keep control over feature set. But, yes, at some point we might want to switch to `@major` only.